packages/Avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+
+import './avatar.scss';
+
+export type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
```

Comment:
I couldn't find a reference to the 'xlarge' size, do you have an example of its use?

Reply:
In the old code, it seems to be used in TeamMemberHeader
![image](https://user-images.githubusercontent.com/110498164/204322647-414e9de1-40d7-440c-87e8-1e2461d6236f.png)